apps/authentication/routes.py

```diff
@@ -43,6 +46,12 @@ def login():
         # Check the password
         if user and verify_pass(password, user.password):
 
+            # Check if user has confirmed the account
+            if user.is_confirmed is False:
+                return render_template('pages/login.html',
+                                       msg='Please confirm your account before login',
+                                       form=login_form)
+
```

Comment:
I believe this code was left behind! can you please double check if this code is still valid?